### PR TITLE
docs: add commit style guidelines

### LINKED ISSUE #1346: Commit Style Guidelines missing in docs
**Is it a ?**

- [ ] Bug Report

- [ ] Feature Request
    
- [ ] Chore

- [x] Documentation
    
**Description**
   Commit Style Guidelines should be present inside the docs to help new contributors

**Would you like to work on the issue?**
   Yes

### PR DESCRIPTION
**Pull request by GOVINDDIXIT**
Fixes #1346 
Checklist:

- [x] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream development branch.

Changes: [ commitStyle.md is added inside docs ]

### CODE REVIEW

File: docs/commitStyle.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+Use an imperative tone to describe what a commit does, rather than what it did. For example, use fix; not fixed or fixes or fixing.
+
+For example: 
+- fix: Typo in Commit Style guidelines 
+- feat: Update UI of SessionDetailsActivity
+- fix: Remove deprecated methods
+- refactor: API endpoints and JSON assets
+instead of writing the following: 
+- Fixed bug with Y
+- Changing behaviour of X
```
Comment (iamareebjamal) [inline docs/commitStyle.md]: This is not imperative

File: docs/commitStyle.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+Use an imperative tone to describe what a commit does, rather than what it did. For example, use fix; not fixed or fixes or fixing.
+
+For example: 
+- fix: Typo in Commit Style guidelines 
+- feat: Update UI of SessionDetailsActivity
+- fix: Remove deprecated methods
+- refactor: API endpoints and JSON assets
+instead of writing the following: 
+- Fixed bug with Y
```
Comment (iamareebjamal) [inline docs/commitStyle.md]: This is not imperative

File: docs/commitStyle.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+The subject is a single short line summarising the change. It should be no greater than 50 characters, should begin with a capital letter and do not end with a period.
+
+Use an imperative tone to describe what a commit does, rather than what it did. For example, use fix; not fixed or fixes or fixing.
+
+For example: 
+- fix: Typo in Commit Style guidelines 
+- feat: Update UI of SessionDetailsActivity
+- fix: Remove deprecated methods
+- refactor: API endpoints and JSON assets
+instead of writing the following:
```
Comment (iamareebjamal) [inline docs/commitStyle.md]: Look at how this is rendered

Comment (GOVINDDIXIT) [reply docs/commitStyle.md]: Rendering fine
https://github.com/GOVINDDIXIT/open-event-orga-app/blob/development/docs/commitStyle.md

Comment (GOVINDDIXIT) [reply docs/commitStyle.md]: @iamareebjamal  pls review 😅

**Comment by GOVINDDIXIT** [comment]
@sridharjajoo pls review

**Comment by iamareebjamal** [comment]
Where is the doc linked?

**Comment by iamareebjamal** [comment]
If it is not linked anywhere, it might as well not be present as nobody can reach to this guideline

File: README.md [hunk 4]
```diff
@@ -139,6 +139,16 @@ So, please take a look at already written tests(they are fairly self-documenting
 
 Lastly, each class should only perform one task, do it well, and be unit tested for it. For example, if a presenter is doing more than it should, i.e., parsing dates or implementing search logic, better move it in its own class. There can be exceptions for this practice, but if the functionality can be generalised and reused, it should most definitely be transferred in its own class and unit tested.
 
+### Code practices
```
Comment (iamareebjamal) [inline README.md]: Change to PR guidelines

**Pull request merged**